src/Compilers/Test/Core/Compilation/ControlFlowGraphVerifier.cs

```diff
@@ -824,8 +830,11 @@ PooledHashSet<CaptureId> adjustAndGetEntryState(ArrayBuilder<PooledHashSet<Captu
                                      isIncrementedNullableForToLoopControlVariable(reference) ||
                                      isConditionalAccessReceiver(reference) ||
                                      isCoalesceAssignmentTarget(reference) ||
-                                     isObjectInitializerInitializedObjectTarget(reference)) &&
-                                 block.EnclosingRegion.EnclosingRegion.CaptureIds.Contains(id)),
+                                     isObjectInitializerInitializedObjectTarget(reference) ||
+                                     isInterpolatedStringArgumentCapture(reference) ||
```

Comment:
>isInterpolatedStringArgumentCapture [](http://example.com/codeflow?start=37&length=35)

It looks like the reference for the special out argument shouldn't be the subject for the relaxed check. It feels like the reference should belong to the enclosing region.